src/level.js

```diff
@@ -12,6 +13,7 @@ const UNITS_WIDE = findAppropriateWidth(UNITS_TALL);
 export default class Level {
     constructor(levelFile) {
         this.entities = [];
+        this.physics = new Physics([UNITS_WIDE, UNITS_TALL]);
```

Comment:
Figured you know this, but you'll have to pass in the dimensions of the level.  `UNITS_WIDE` and `UNITS_TALL` is just the viewport dimensions.